src/server/server.go

```diff
@@ -1,21 +1,29 @@
 package server
 
 import (
-	"github.com/gofiber/fiber/v2"
-	"gorm.io/gorm"
+	"onepixel_backend/src/controllers"
 	"onepixel_backend/src/routes/api"
+
+	"gorm.io/gorm"
+
+	"github.com/gofiber/fiber/v2"
 )
 
-func CreateApp(db *gorm.DB) *fiber.App {
+func CreateApp(dbConn *gorm.DB) *fiber.App {
 	app := fiber.New()
 
+	usersController := controllers.NewUsersController(dbConn)
+	
 	app.Get("/", func(c *fiber.Ctx) error {
 		return c.SendString("Hello, World 👋!")
 	})
 
 	apiV1 := app.Group("/api/v1")
 
-	apiV1.Route("/users", api.UsersRoute(db))
+	apiV1.Route("/users", func(router fiber.Router) {
```

Comment:
would prefer these to remain 1-line and let the handling happen inside `routes/api/users.go` as I had implemented